app/models/orchestration_stack.rb

```diff
@@ -14,6 +14,7 @@ class OrchestrationStack < ApplicationRecord
 
   belongs_to :ext_management_system, :foreign_key => :ems_id
 
+  has_many   :authentications, :through => :authentication_orchestration_stacks
```

Comment:
you need to also define 

has_many :authentication_orchestration_stacks

otherwise calling OrchestrationStack.first.authentications throws an exception